pkg/ebpf/c/tracee.bpf.c

```diff
@@ -655,6 +658,234 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
     return 0;
 }
 
+void update_hidden_modules(u64 mod_addr, char *mod_name)
```

Comment:
i think that the method that was used to detect the hidden module is valuable.
for example, adding an argument here that states the method (procfs / kset / tree) and add it to the `kernel_module_t` key.

then in user space you can add it as an argument to the event, or create 3 different derived events.

WDYT?

note: if you decide to do this, i think that `already_reported_hidden_modules_map` map key should also contain the method. i think it would be valuable to see that kernel module is hidden from procfs, and mod_tree, but not module_kset for example.